docs/tutorial/gpu_user_guide.md

```diff
@@ -0,0 +1,104 @@
+### GPU User Guide
+
+#### Step 1: Prepare the system for NVIDIA GPU support
+
+To enable NVIDIA GPU support, follow these steps to configure the system:
+
+```bash
+distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
+      && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
+      && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | \
+            sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
+            sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+```
+
+#### Step 2: Install NVIDIA Container Toolkit
+
+Use the following command to install the NVIDIA Container Toolkit:
+
+```bash
+sudo apt-get update && \
+sudo apt-get install -y nvidia-container-toolkit && \
+sudo nvidia-ctk runtime configure --runtime=docker
+```
+
+This will install the necessary components for NVIDIA GPU support in Docker, enabling you to utilize GPU resources within Docker containers.
+
+#### Step 3: Set NVIDIA as the default runtime for Docker
+
+To configure Docker to use NVIDIA as the default runtime, follow these steps:
+
+1. Edit the Docker daemon configuration file:
+
```

Comment:
`$ sudo nvidia-ctk runtime configure --runtime=docker --set-as-default` can also be used with configuration in one step.

Reply:
thx